### PR TITLE
PLT-7508: Add settings to disable wide mention confirmation

### PR DESCRIPTION
**Pull request by jespino**
The PR mattermost/mattermost-webapp#18 depends on this commit.

Links
-------
mattermost/mattermost-server#7335
https://mattermost.atlassian.net/browse/PLT-7508

Checks
----------
- [X] Has UI changes
- [X] Includes text changes and localization file

### CODE REVIEW

**Comment by esethna** [comment]
Moving this to dev review as there is nothing to test client-side. We will test the webapp PR once this is merged and we can get a test server running.

File: config/default.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
         "UserStatusAwayTimeout": 300,
         "MaxChannelsPerTeam": 2000,
         "MaxNotificationsPerChannel": 1000,
+        "ConfirmNotificationsToChannel": true,
```
Comment (jwilander) [inline config/default.json]: I know it'll make the setting name a bit longer but we have a convention for prepending settings that enable or disable functionality with `Enable`. Would you be ok making this `EnableConfirmNotificationsToChannel`?

Comment (jespino) [reply config/default.json]: Sure! I'll modify the PR ASAP :smiley:

**Comment by jespino** [comment]
Changed and rebased to master (and soon changed in the mattermost-webapp PR)

**Comment by hmhealey** [comment]
Looks good, @jespino

**Pull request merged**